### DISCUSSION TITLE
Grove-Doppler-Radar | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#59] [Q&A]
# Grove-Doppler-Radar | Seeed Studio Wiki

How would you build a system that could calculate the distance towards an object or detect whether there is motion present? Normally you would use an Ultrasonic or LiDAR Sensor for distance measurement and PIR Motion Sensor for motion detection. What if we told you there is an all-in-one module that could do all these functions more precisely and also perform functions such as velocity detection of moving objects and angle detection of objects. Would you believe it? We were tired of using these traditional modules for motion-sensing applications and wanted to deliver you a better solution, integrating new technologies. Well…For the first time in the history of Grove, we are very excited to bring you a Grove Module based on Radar Technology!

https://wiki.seeedstudio.com/Grove-Doppler-Radar/

**Comment by DanKrones** [comment]
Is there any demo code available to measure the distance to an object?

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.

**Comment by Arexx-Engineering** [comment]
Hello,

Is it possible to switch of the Receiver?
Is it possible to put bthe transmitter on continues wave.

Appriciate any help on this, many thanks in advance.

Regards,

Aalt Rens
